custom/icds_reports/static/js/directives/help-popup/help-popup.directive.js

```diff
@@ -4,8 +4,9 @@ function HelpPopupController($scope) {
     $scope.activeInfoHeading = '';
     $scope.activeInfoHelp = '';
     function showHelp(heading, help) {
-        $scope.activeInfoHeading = heading;
-        $scope.activeInfoHelp = help;
+        var regex = /(<([^>]+)>)/ig;
+        $scope.activeInfoHeading = heading.replace(regex, "");
+        $scope.activeInfoHelp = help.replace(regex, "");
```

Comment:
It would likely be better to use the `html` binding than to hackily replace these via regex. See here: https://github.com/dimagi/commcare-hq/pull/26289/files for one implementation of that.

Reply:
So this is already done. I will just undo the changes.